### PR TITLE
Feature/22

### PR DESCRIPTION
**Pull request by ChrisCottier**
Finished creating navigation element of driver registration. Button design do not match design docs, but the functionality is there.

### CODE REVIEW

File: web-client/src/shared/components/drivers-registration/CurrentSlide/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import CurrentSlide from './index';
+
+describe('Current Slide', () => {
+  test('should render Personal Info slide first.', () => {
+    const { getByText } = render(<CurrentSlide currentSlide={1}/>);
+
+    const firstNameLabel = getByText('First Name');
+
+    expect(firstNameLabel).toBeInTheDocument;
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/CurrentSlide/index.test.tsx:12]: Awesome!

File: web-client/src/shared/components/drivers-registration/CurrentSlide/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import PersonalInfo from '../PersonalInfo/index'
+
+interface Props {
+    currentSlide: number
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/CurrentSlide/index.tsx:6]: perfect

File: web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx:2]: Perfect

**Review by l2silver — COMMENTED** [review]
Hey, it looks like there are some lint warnings. I just changed the lint script to throw an error, but in the mean time, can you fix the lint warnings. And I'll try and figure out why netlify was angry.

**Comment by ChrisCottier** [comment]
> Hey, it looks like there are some lint warnings. I just changed the lint script to throw an error, but in the mean time, can you fix the lint warnings. And I'll try and figure out why netlify was angry.

Added return types to React function components. Lint warnings are gone now.

File: web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    numberOfSlides: number,
+    nextSlide(event: React.MouseEvent): void,
+    prevSlide(event: React.MouseEvent): void,
+}
+
+const NavigationButtons = ({currentSlide, numberOfSlides, nextSlide, prevSlide}: Props): React.ReactElement => {
+
+    return (
+        <>
+            <ButtonGroup size="large">
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx:16]: Perfect use of button group!

File: web-client/src/shared/components/drivers-registration/CurrentSlide/index.test.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import CurrentSlide from './index';
+
+describe('Current Slide', () => {
+  test('should render Personal Info slide first.', () => {
+    const { getByText } = render(<CurrentSlide currentSlide={1}/>);
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/CurrentSlide/index.test.tsx:8]: Yes!!! Thank you for testing current slide and not just PersonalInfo.

**Review by l2silver — CHANGES_REQUESTED** [review]
Sorry Chris, 1 more change and we're good.

**Comment by ChrisCottier** [comment]
> Sorry Chris, 1 more change and we're good.

Sorry I'm not as familiar with GitHub reviews, I couldn't see what / where you wanted changes. Or is it related to the newlines at the end of the files?

File: web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+                {currentSlide > 1 ?
+                <Button
+                color="secondary"
+                variant="contained"
+                onClick={prevSlide}
+                >Back
+                </Button>
+                :<></> }
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx:24]: Do you think you could change instances 
condition ? Component : <></>
to
condition && Component
for when condition is boolean
and condition ? Component : null
when condition has the possibility of being the number 0

**Comment by l2silver** [comment]
Hey Chris, sorry my comment wasn't recorded. It's this one: https://github.com/diglit/volunteer-toronto-volunteer-drivers-app/pull/27#discussion_r559697576

File: web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx (hunk 6 above)
Comment (ChrisCottier) [reply web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx:24]: Does that mean just change the React fragments to null instead? Or did you want me to change the turnery logic as well?

Comment (ChrisCottier) [reply web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx:24]: Just tried it out- changing the fragments to null got rid of the material ui warning.

Comment (l2silver) [reply web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx:24]: I think that's a good thing, no?

Comment (ChrisCottier) [reply web-client/src/shared/components/drivers-registration/NavigationButtons/index.tsx]: Totally, was just making sure I followed. New commit just pushed to this branch.

**Review by l2silver — APPROVED** [review]

**Review by OksanaSam — APPROVED** [review]
Looks really nice, great job Chris!

**Review by WendyChenj — APPROVED** [review]

**Pull request merged**